sdk/storage/blobs/src/az_storage_blobs_blob_client.c

```diff
@@ -26,8 +27,9 @@ enum {
 
 az_storage_blobs_blob_client_options const AZ_STORAGE_BLOBS_BLOB_CLIENT_DEFAULT_OPTIONS
     = { .retry = {
-            .max_retry = 5,
-            .delay_in_ms = 1000,
+            .max_tries = 5,
```

Comment:
max_retries

Reply:
FWIW there doesn't seem to be a standard terminology for the retry policies.  Unfortunate that each language has its own terminology for what is the same thing @JeffreyRichter

.NET
```
private readonly TimeSpan _delay;
private readonly TimeSpan _maxDelay;
private readonly int _maxRetries;
```
JS
```
retryCount?: number,
retryInterval?: number,
maxRetryInterval?: number
```
Python
```
'total': options.pop("retry_total", self.total_retries),
'connect': options.pop("retry_connect", self.connect_retries),
'read': options.pop("retry_read", self.read_retries),
'status': options.pop("retry_status", self.status_retries),
'backoff': options.pop("retry_backoff_factor", self.backoff_factor),
'max_backoff': options.pop("retry_backoff_max", self.BACKOFF_MAX),
'methods': options.pop("retry_on_methods", self._method_whitelist),
'history': []
```
```
    /**
     * Max number of retry attempts to be make.
     *
     * @return The max number of retry attempts.
     */
    int getMaxRetries();

    /**
     * Computes the delay between each retry.
     *
     * @param retryAttempts The number of retry attempts completed so far.
     * @return The delay duration before the next retry.
     */
    Duration calculateRetryDelay(int retryAttempts);
```

Reply:
Yes, the inconsistency is unfortunate. I think we should use:
max_retries
retry_delay_msec
max_retry_delay_msec